tests/models/VegetableTranslation.php

```diff
@@ -7,4 +7,10 @@
 class VegetableTranslation extends Eloquent
 {
     public $timestamps = false;
+
+    protected $fillable = [
+        'country_id',
```

Comment:
This should be `vegetable_identity` or something like that if I recall correctly 🤔

Reply:
https://github.com/Astrotomic/laravel-translatable/pull/76/commits/5d352596e295a48d38a17323e6b839f3e4221e38